tests/plugins/test_build_source_container.py

```diff
@@ -187,7 +187,8 @@ def check_check_output(args, **kwargs):
         build_result = runner.run()
         err_msg = "No SRPMs directory '{}' available".format(sources_dir_path)
         err_msg += "\nNo Remote source directory '{}' available".format(remote_dir_path)
-        assert err_msg in caplog.text
+        # Since Python 3.7 logger adds additional whitespaces by default -> checking without them
+        assert err_msg.replace(" ", "") in caplog.text.replace(" ", "")
```

Comment:
The error message remains the same, but we create new strings without white spaces and check them, so it shouldn't make misunderstandings

Reply:
What I mean here is that, if the assertion fails, the error message will be harder to read for the reader.

For instance, we would see something like this:

```
AssertionError: assert 'NoSRPMsdirectory/sourcesavailable\nNoRemotesourcedirectory/remotesourcesavailable' in NoSRPMsdirectory/sourcesavailable\nNoRemotesourcedirectory/remotesourceavailable'
```

This is also error prone since it is verifying a modified version of the stings and not the actual strings as you said. We are creating new strings and checking the newly created strings instead of checking whatever output comes from the plugin being tested.

Reply:
@athos-ribeiro You`re right, I didn`t understand you :) 
It will be better, if I change those messages by regex.